### DISCUSSION TITLE
TooltipTriggerProps should be exported

### DISCUSSION THREAD

**Discussion by brandonpittman** [adobe/react-spectrum#3343] [Ideas]
https://github.com/adobe/react-spectrum/blob/02d3bd712a2020db5cad031526f948e0644c1018/packages/%40react-types/tooltip/src/index.d.ts#L17

I'm building a custom tooltip and having an exported `TooltipTriggerProps` type would be really helpful.

**Comment by LFDanLu** [comment]
Definitely a reasonable ask, https://github.com/adobe/react-spectrum/pull/3326/files#diff-1870428555f21c08d34188634fbc9c7ffc8f4db8977420f38b4efa9b3c9b7581 will handle exporting that type.
